src/libawkward/builder/ArrayBuilder.cpp

```diff
@@ -1,493 +1,411 @@
-// BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+// BSD 3-Clause License; see
+// https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
 
-#define FILENAME(line) FILENAME_FOR_EXCEPTIONS("src/libawkward/builder/ArrayBuilder.cpp", line)
+#define FILENAME(line)                                                         \
+  FILENAME_FOR_EXCEPTIONS("src/libawkward/builder/ArrayBuilder.cpp", line)
```

Comment:
Much like the two-line LICENSE comment, this multi-line split is really unnecessary. It might be harder to explain to clang-format that we want it off, though.